src/Resources/Resources.Test/ScenarioTests/DeploymentScriptsTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.Resources.Test.ScenarioTests
+{
+    public class DeploymentScriptsTests : ResourceTestRunner
+    {
+        public DeploymentScriptsTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact()]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestDeploymentScriptGetForPowerShell()
+        {
+            TestRunner.RunTestScript("Test-GetDeploymentScript-PowerShell");
+        }
+
+        [Fact()]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestDeploymentScriptGetForCli()
+        {
+            TestRunner.RunTestScript("Test-GetDeploymentScript-Cli");
+        }
+
+        [Fact(Skip = "Need to wait until the Resources .NET SDK is updated to 3.3.0 in the PS module.")]
```

Comment:
When do you plan to update the SDK

Reply:
There will be a separate PR after we have this version of the cmdlets checked in. 

---
In reply to: [390161237](https://github.com/Azure/azure-powershell/pull/11266#discussion_r390161237) [](ancestors = 390161237)